airflow/ui/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ui",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
+    "build": "webpack --mode production",
+    "test": "jest",
+    "lint": "eslint --format codeframe --ext mjs,jsx,js,tsx,ts src test && tsc"
+  },
+  "dependencies": {
+    "@chakra-ui/react": "^1.3.4",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.1.5",
+    "dotenv": "^8.2.0",
+    "framer-motion": "^3.10.0",
+    "prop-types": "^15",
```

Comment:
Is this actually needed?

Reply:
Good catch. Cleaned up a few other libraries we don't need at the moment.